keyboards/gray_studio/space65/keymaps/billiams/keymap.c

```diff
@@ -0,0 +1,125 @@
+   * | Shift     |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  | ?/ |    Shift   | up  |  ESC |
+   * |------------------------------------------------------------------------------------------------+
+   * | Ctrl |  Alt  |  Cmd  |              Space               |  Alt |  Ctrl |  Left  | Down | Right |
+   * `------------------------------------------------------------------------------------------------'
+   */
+
+[0] = LAYOUT( \
+        KC_GRAVE, KC_1, KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_DEL, KC_INS, \
+        KC_TAB,   KC_Q, KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC, KC_PGUP,          \
+        MO(1),    KC_A, KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGDN, \
+        KC_LSFT, KC_LSFT, KC_Z, KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_ESC,         \
+        KC_LCTL, KC_LALT, KC_LGUI,                KC_SPC, KC_SPC, KC_SPC,         KC_RALT, KC_RCTL,   KC_LEFT, KC_DOWN, KC_RGHT                   \
+    ),
+
+  /* 1st Layer
+   * ,------------------------------------------------------------------------------------------------.
+   * |     |  F1 |  F2 |  F3 |  F4 |  F5 |  F6 |  F7 |  F8 |  F9 | F10 | F11 | F12 |Vol- |Vol+ | Next |
+   * |------------------------------------------------------------------------------------------------+
+   * |        |RGB T|RGB M| Hue-| Hue+| Sat-| Sat+| Val-| Val+|      |      |    |      |      | Prev |
+   * |------------------------------------------------------------------------------------------------+
+   * |         |     |     |     |     |     | Left| Down| Up |Right|      |      | Play/Pause | Vol+ |
+   * |------------------------------------------------------------------------------------------------+
+   * |           |     |     |     |     |     |     |     |    | Scr- | Scr+ |    |     |PGUP | Vol- |
+   * |------------------------------------------------------------------------------------------------+
+   * |      |       |       |                                  |      |  Fn2  |  HOME  | PGDN  |  END |
+   * `------------------------------------------------------------------------------------------------'
+   */
+
+[1] = LAYOUT( \
+             KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS, KC_MFFD, \
+             KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MRWD,          \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_UP, KC_RIGHT, KC_TRNS, KC_TRNS, KC_MPLY, KC_VOLU,                   \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BRID, KC_BRIU, KC_TRNS, KC_TRNS, KC_PGUP, KC_VOLD, \
```

Comment:
The `BL_TOGG` keycode described in your readme isn't present.

```suggestion
             KC_TRNS, KC_TRNS, BL_TOGG, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BRID, KC_BRIU, KC_TRNS, KC_TRNS, KC_PGUP, KC_VOLD, \
```

Reply:
@noroadsleft I actually wanted to remove the BL_* mappings.